src/components/Price.test.js

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from '@testing-library/react'
+import Price from './Price'
+
+test('renders price component with data received in props', () => {
+  render(
+    <Price price="50000.00" time="Monday June 7 2021, 12:00:00 PM" symbol="€" />
+  )
+  expect(
+    screen.getByText('as at Monday June 7 2021, 12:00:00 PM')
```

Comment:
Use `queryBy...` when trying to find the existence to something.

https://testing-library.com/docs/queries/about/#types-of-queries

With `getBy...`, it will throw an error but with `queryBy...`, it will actually check this assertion.

Also see https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence (creator of RTL)

Reply:
After discussion, decided that `getBy` would be more appropriate in this test situation. No changes made.